source/includes/harvest/_demographic_data.md

```diff
@@ -47,8 +73,9 @@ Demographic questions and answers submitted during the application process. For
 | Attribute | Description |
 |-----------|-------------|
 | id | The demographic answer option's unique identifier |
-| translations.language | Only `en` (English) is supported at this time.
-| translations.name | The answer option text.
+| name | The answer option text
+| translations.language | Translations have been deprecated.  Only `en` (English) is supported at this time.
+| translations.name | Translations have been deprecated. This value will be the same as `name` above.
```

Comment:
Update to "Translations have been deprecated but are kept for backwards compatibility."